socialdistribution/bettersocial/views.py

```diff
@@ -154,7 +154,23 @@ def get_context_data(self, **kwargs):
             context['posts'] = author.post_set.all()
         else:
             context['author_following_user'] = bool(Following.objects.filter(author = author_uuid, following_uuid = user_uuid))
-            context['user_following_author'] = bool(Following.objects.filter(author = user_uuid, following_uuid = author_uuid))
+            if Author.objects.filter(uuid = user_uuid).exists():
+                context['user_following_author'] = bool(Following.objects.filter(author = user_uuid, following_uuid = author_uuid))
+            else:
+                # TODO: make sure this works
+                node = Node.objects.filter(host__contains = self.request.GET['host']).get()
+                url = yarl.URL(node.host + '/author/' + author_uuid + '/followers/' + user_uuid).human_repr()
```

Comment:
You should use yarl's overloaded methods for building the URL. This prevents trailing slashes from collecting, and you also don't have to worry about them either.

```suggestion
                url = (yarl.URL(node.host) / 'author' / author_uuid / 'followers' / user_uuid / '').human_repr()

```

Reply:
right yea I noticed earlier today and changed this in another branch. Thanks